sdk/loadtestservice/Azure.Developer.LoadTesting/tests/LoadTestingClientTest.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Azure.Developer.LoadTesting.Tests
+{
+    public class LoadTestingClientTest: RecordedTestBase<LoadTestingClientTestEnvironment>
+    {
+        public string testId;
+        public string fileId;
+        public string testRunId;
+        public string appComponentId;
+
+        public LoadTestingClientTest(bool isAsync) : base(isAsync, RecordedTestMode.Record)
```

Comment:
Based on the build errors, it looks like you need a body sanitizer for the storage sas tokens

here is an example: 
https://github.com/Azure/azure-sdk-for-net/blob/bd5c715e0b25f80fee7e78f779c1cab9f6387f50/sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs#L34-L36